system-addon/lib/ActivityStream.jsm

```diff
@@ -25,12 +25,13 @@ const {TopStoriesFeed} = Cu.import("resource://activity-stream/lib/TopStoriesFee
 const REASON_ADDON_UNINSTALL = 6;
 
 // Sections, keyed by section id
+const showTopStoriesByDefault = ["en-US", "en-CA"].includes(Services.locale.getRequestedLocale());
 const SECTIONS = new Map([
   ["topstories", {
     feed: TopStoriesFeed,
     prefTitle: "Fetches content recommendations from a configurable content provider",
      // for now, we only want to show top stories by default to the following locales
-    showByDefault: ["en-US", "en-CA"].includes(Services.locale.getRequestedLocale())
+    showByDefault: showTopStoriesByDefault
```

Comment:
Keep the default inline with topstories, i.e., undo this change. So that the comment line makes sense.